src/pubtools/_marketplacesvm/cloud_providers/base.py

```diff
@@ -151,7 +151,7 @@ def _delete_push_images(
         self,
         push_item: T,
         **kwargs,
-    ) -> T:
+    ) -> Tuple[T, Any]:
```

Comment:
Could you add also missing Returns section to the docuemtnation

Reply:
Added